### PR TITLE
Install dependencies for scripts

### PR DESCRIPTION
**Pull request by jburel**
The dependencies were not included in the literal include when the files got re-organised
leading to confusion 
see for example https://www.openmicroscopy.org/community/viewtopic.php?t=8383&p=18639#p18639
cc @kennethgillen @hflynn

### CODE REVIEW

**Comment by hflynn** [comment]
Sections are present in the merge build docs now, ready to be checked

**Comment by jburel** [comment]
I need to review the pages web with server too before review

**Comment by kennethgillen** [comment]
Nice one, thanks.

File: omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6.rst [hunk 1]
```diff
@@ -18,12 +18,6 @@ Installing prerequisites
 
 **The following steps are run as root.**
 
-Install Pillow and Numpy:
```
Comment (joshmoore) [inline omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6.rst:21]: Why are some blocks removed?

Comment (jburel) [reply omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6.rst:21]: b/c they are already installed in the server page and this is the doc for web with server

**Pull request merged**